loleaflet/src/control/Control.UserList.js

```diff
@@ -101,7 +102,27 @@ L.Control.UserList = L.Control.extend({
 		return content;
 	},
 
-	updateUserListCount: function() {
+	renderUserAvatars: function(viewId, userName, extraInfo, color) {
+		if (extraInfo !== undefined && extraInfo.avatar !== undefined) {
+			var avatarElement = '<div class="user-item-wrapper" id="user-wrapper-' + viewId + '"><p class="user-item" id="user-' + viewId 
+			+ '" style="background-image:url(\'' + extraInfo.avatar + '\');border-color:' + color + '"></p><span> '+ userName +'</span></div>';
+			if ($('#follow-editor').length == 0) {
+				$('#listActiveUser').append(avatarElement);
```

Comment:
The general naming is a bit hard to follow, maybe stick to something like this for the elements:

- userListSummary: For the avatars shown in the header (those should be limited to a maximum number, more users will then be shown in the popover)
- userListPopover: Shows a list of all users

That naming would ideally be reflected in the css id/class names as well as the variables that contain the HTML elements created.